domains/wyatt.json

```
@@ -0,0 +1,11 @@
+{
+    "description": "Blog",
+    "repo": "https://github.com/ws998116/ws998116.github.io",
+    "owner": {
+        "username": "ws998116",
+        "email": "ws998116@gmail.com"
+    },
+    "record": {
+        "CNAME": "https://ws998116.github.io/"
```

CNAME can't contain protocol or `/`
```suggestion
        "CNAME": "ws998116.github.io"
```